# 1.60
Vi slepper en ball fra en 20m høy bygning slik at farten bli så stor at vi ikke kan se bort fra luftmotsand. Akselerasjonen til ballen er $a(v)=-9,81m/s^2 + 0,034m^(-1) \cdot v^2$

### a) Lag et program som simulerer bevegelsen, og som finner ut hvor lang tid ballen bruker før den treffer bakken

In [24]:
s = 20
v = 0.0
t = 0.0

s_slutt = 0
dt = 0.001

def a(v):
    return -9.81 + 0.034 * v**2

while s > s_slutt:
    v += a(v)*dt
    s += v*dt
    t += dt

print(f"Det tar {round(t, 1)}s før ballen treffer bakken")

Det tar 2.3s før ballen treffer bakken


### b) Samenlign svaret ditt i opgave a med tide ballen ville ha brukt på å treffe bakken dersom vi hadde rekna med fritt fall.

$v=\sqrt{2as-v_0}$

$t=\frac{v-v_0}{a}=\frac{\sqrt{2as-v_0}-v_0}{a}=-2.2$

2s

### c) Finn høyden med og uten luftmotsand som gir samme svar med to gjeldene siffer

Vi vet at den bruker $2.3s$ med luftmotstand, så trenger vi bare å finne en strekning som bruker $2,3s$

$S=v_0t+\frac{1}{2}at^2=\frac{1}{2} \cdot -9.81m/s^2 \cdot 2.3^2m^2/s^2=-26m$

så $26m$ uten luftmotstand og $20m$ med luftmotstand vil treffe samtidig